src/app/templates/app/license_requests.html

```diff
@@ -46,6 +46,9 @@
 
 <style>
     th {text-align: center;}
+    .clickableRow {cursor: pointer;}
+    table#requests_table.dataTable td:hover {background-color: #ddd;}
+    table#requests_table.dataTable .bt:hover {background-color: #f9f9f9;}
```

Comment:
Same for these one

Reply:
Moved `<style>` to base.html, used css for `tr` tag without id.